apps/mull-ui/src/app/components/event-bullet/event-bullet.tsx

```diff
@@ -9,21 +9,22 @@ export interface EventBulletProps {
   onClick?: (event: React.MouseEvent<HTMLInputElement>) => void; //TODO: redirect to the correct DM page
 }
 
-export const EventBullet = ({ eventTitle, eventPicture, eventDate }: EventBulletProps) => {
+export const EventBullet = ({ eventTitle, eventPicture, eventDate, onClick }: EventBulletProps) => {
```

Comment:
💡Event Bullet looks really ugly to me. Not really your fault since you're just following the mockup. I don't know if you agree.

![image](https://user-images.githubusercontent.com/23544999/113075988-1b11a900-919c-11eb-800a-d3f2a5312111.png)

Id say some easy changes to do would be to:
- Add a border radius to the containerso the highlight isn't square
- Set the max-height of the image to be 100% and give a big increase to the max-width, so that the image will be the same height as the date square in most cases.

Reply:
yeah I agree it doesn't look great, thanks for the suggestion :) We will work on that

Reply:
Oh yea i was the one who made this, yeah i think i only took perfectly square pictures in the events when i was testing so it had looked good to me then. +1 on the suggestions

Reply:
Yeah I see that, square images work better, but other ratios not so much